CHANGELOG.md

```diff
@@ -10,10 +10,16 @@ Current
 
 ### Added:
 
+- [Support for Druid's In Filter](https://github.com/yahoo/fili/pull/64)
```

Comment:
If it's included, sweet. But that means what we should just indicate that it's available in v0.9.0 instead o v0.9.1 :-P